src/main/java/bio/overture/ego/repository/queryspecification/builder/AbstractSpecificationBuilder.java

```diff
@@ -23,10 +22,10 @@
     };
   }
 
-  public Specification<T> listAll(){
+  public Specification<T> listAll() {
     return (fromUser, query, builder) -> {
       val root = setupFetchStrategy(fromUser);
-      return builder.isNotNull(root.get(JavaFields.ID));
+      return null;
```

Comment:
Is this because we are expecting an override on this method?

Reply:
@andricDu  after Jon and I investigated how spring-jpa builds the query under, we found an if statement that excludes the where clause if the `Predicate` returned from the `Specification` returns null. The only way I knew how to do this before learning this, was to perform a null check on the Id, which is always non-null.